src/python/pants/backend/python/typecheck/pyright/rules_integration_test.py

```diff
@@ -152,21 +152,21 @@ def test_failing(rule_runner: PythonRuleRunner) -> None:
 def test_multiple_targets(rule_runner: PythonRuleRunner) -> None:
     rule_runner.write_files(
         {
-            f"{PACKAGE}/good.py": GOOD_FILE,
-            f"{PACKAGE}/bad.py": BAD_FILE,
+            f"{PACKAGE}/bad1.py": BAD_FILE,
```

Comment:
There are tests covering "good" files, so that's alright that both of the files in this scenario are "bad".